web/client/reducers/map.js

```diff
@@ -67,14 +67,13 @@ function mapConfig(state = null, action) {
         case ZOOM_TO_EXTENT: {
             let zoom = 0;
             let bounds = CoordinatesUtils.reprojectBbox(action.extent, action.crs, state.bbox && state.bbox.crs || "EPSG:4326");
-            let wgs84BBounds = CoordinatesUtils.reprojectBbox(action.extent, action.crs, "EPSG:4326");
-            if (bounds && wgs84BBounds) {
+            if (bounds) {
                 // center by the max. extent defined in the map's config
-                let center = MapUtils.getCenterForExtent(wgs84BBounds, "EPSG:4326");
+                let center = CoordinatesUtils.reproject(MapUtils.getCenterForExtent(action.extent, action.crs), action.crs, 'EPSG:4326');
```

Comment:
that's because action.extent is an array of strings and MapUtils.getCenterForExtent requires an array of numbers.
going to fix